public/assets/css/header.css

```diff
@@ -1,13 +1,48 @@
-body > header > nav > ul > li:first-child {
+.icon {
```

Comment:
create a file path like this `css/icons/home.css` and  `css/icons/icons.css`

in `css/icons/icons.css` would contain a global rule of the articles and would import the other files, like this :

in `css/icons/icons.css`
```css
.icon-circle{
...
}
.icon-circle:hover{
...
}
```
and `css/icons/home.css`
```css
.icon-circle .home{
...
}
.home-xxx{
...
}
```
or 
```css
.icon-circle .icon-home{
...
}
.icon-home-xxx{
...
}
```

Reply:
"xxx" rename like this exemple : "part-one", ...

```css
/* bem */
.icon-email-part--one{
...
}

```
or 
```css
/* oocss */
.icon-email-part-one{
...
}

```